project-templates/import-records/Readme

```diff
@@ -0,0 +1,122 @@
+Requirements
+
+  node >= 4 https://nodejs.org/
+  make      https://www.gnu.org/software/make/
+  jq        https://stedolan.github.io/jq/
+
+Make does not support filenames with special characters unfortunately.  Remove
+special characters in your filenames.
+
+Make is not included on OSX by default, you need to install XCode or use a
+package manager like [Brew](https://brew.sh/) or
+[Nix](https://nixos.org/nix/).
```

Comment:
Aside: I think regardless you need Xcode right? It's just that brew does it for you if you haven't done it manually

(nothing says "pro computer" like having to download 3 gigs of crap just to get gcc)